content/validation/before/domains.md

```diff
@@ -0,0 +1,39 @@
+Data plays a crucial role in the process of training any ML model. It is something from which the model learns to solve a given task and therefore care needs to be taken with its handling:
+
+1. By making sure that a data set is relevant to the problem and no deviation from expectations is introduced: so that a resulted model doesn't incorporate these undesired discrepancies.
```

Comment:
"introduced. This makes sure that the model doesn't incorporate undesired fluctuations or features."